src/app/api/tasks/tasks.service.ts

```diff
@@ -532,4 +538,14 @@ export class TasksService extends BaseService {
     // --- Handle task moved to complete
     await notificationService.markAsReadForAllRecipients(updatedTask)
   }
+
+  async getSignedUrl(filePath: string) {
+    const supabase = new SupabaseService()
+    const { data, error } = await supabase.supabase.storage.from(supabaseBucket).createSignedUrl(filePath, signedUrlTtl)
+    if (error) {
+      throw new APIError(httpStatus.BAD_REQUEST)
+    }
```

Comment:
Error handling is improper. This way the error will be silenced and nothing will appear in our logs.

Please refer to proper usage for APIError for this, unless suppressing the error is what you're going for.